src/plugins/intel_gpu/src/plugin/infer_request.cpp

```diff
@@ -110,6 +110,9 @@ void checkInputBlob(const Blob::Ptr &blob,
     bool nv12_two_inputs = false) {
     const std::string strNotMatched("The input blob size is not equal to the network input size");
 
+    if (foundInput->getInputData()->isDynamic()) {
+        return;
+    }
```

Comment:
deprecated API, don't use it

Reply:
Use API from here https://github.com/openvinotoolkit/openvino/pull/9557

Reply:
Fixed